docs/dev-guide/iframe.md

```diff
@@ -306,465 +359,170 @@ api.executeCommand('sendTones', {
-```javascript
-api.removeEventListeners([ 'incomingMessage', 'outgoingMessageListener' ]);
-```
-
-You can get the number of participants in the conference with the following API function:
-```javascript
-const numberOfParticipants = api.getNumberOfParticipants();
-```
-
-You can get the avatar URL of a participant in the conference with the following API function:
-```javascript
-const avatarURL = api.getAvatarURL(participantId);
-```
+* **stopRecording** - Stops an ongoing **`stream`** or **`file`** recording.
 
-You can get the display name of a participant in the conference with the following API function:
-```javascript
-const displayName = api.getDisplayName(participantId);
-```
+  No other params are required.
 
-You can get the email of a participant in the conference with the following API function:
 ```javascript
-const email = api.getEmail(participantId);
+api.executeCommand('stopRecording', 
+    mode: string //recording mode to stop, `stream` or `file`
+);
 ```
 
-You can get the iframe HTML element where Jitsi Meet is loaded with the following API function:
-```javascript
-const iframe = api.getIFrame();
-```
+## **Multiple command execution**
 
-You can check whether the audio is muted with the following API function:
-```javascript
-api.isAudioMuted().then(muted => {
-    ...
-});
-```
+You can also execute multiple commands using the **`executeCommands`** method:
 
-You can check whether the video is muted with the following API function:
 ```javascript
-api.isVideoMuted().then(muted => {
-    ...
-});
-```
-
-You can check whether the audio is available with the following API function:
-```javascript
-api.isAudioAvailable().then(available => {
-    ...
-});
+api.executeCommands(commands);
 ```
 
-You can check whether the video is available with the following API function:
-```javascript
-api.isVideoAvailable().then(available => {
-    ...
-});
-```
+The **`commands`** parameter is an object with the names of the commands as keys and the arguments for the commands as values:
 
-You can invite new participants to the call with the following API function:
 ```javascript
-api.invite([ {...}, {...}, {...} ]).then(() => {
-    // success
-}).catch(() => {
-    // failure
+api.executeCommands({
+    displayName: [ 'nickname' ],
+    toggleAudio: []
 });
 ```
-**NOTE: The format of the invitees in the array depends on the invite service used for the deployment.**
-
-You can remove the embedded Jitsi Meet Conference with the following API function:
-```javascript
-api.dispose();
-```
 
-NOTE: It's a good practice to remove the conference before the page is unloaded.
+## **EventEmitter**
 
-[config.js]: https://github.com/jitsi/jitsi-meet/blob/master/config.js
-[interface_config.js]: https://github.com/jitsi/jitsi-meet/blob/master/interface_config.js
-[EventEmitter]: https://nodejs.org/api/events.html
+Jitsi Meet incorporates use of [EventEmitter](https://angular.io/api/core/EventEmitter
```

Comment:
The proper link is  https://nodejs.org/api/events.html

Reply:
UPdated